requirements.txt

```diff
@@ -17,3 +17,4 @@ pydot>=1.0.28,<=1.0.29
 psutil>=1.2.1,<=3.4.2
 matplotlib>=1.3.1,<=1.5.1
 scikit-fmm>=0.0.9
+python-magic>=0.4
```

Comment:
@TimZaman can we get away without using magic? Is this the imagemagick wrapper? Any idea which deb package it would correspond to? https://packages.ubuntu.com/search?keywords=python+magic